modsettings/CustomUIAssets.json

```diff
@@ -75,13 +75,23 @@
     "URL": "http://cloud-3.steamusercontent.com/ugc/2027213118466516005/113C19D37CFFA9E554394FD5B11B32967F846A62/"
   },
   {
-    "Name": "option_image1",
+    "Name": "option_header1",
```

Comment:
Better to name these what they are, and then they just get used.  Maybe like "header_core" and "header_acolyte"?

Reply:
Changed in part 5 to avoid conflict